tests/rptest/services/redpanda.py

```diff
@@ -3634,57 +3639,69 @@ def check():
 
         return wait_until_result(check, timeout_sec=30, backoff_sec=1)
 
-    def stop_and_scrub_object_storage(self):
-        # Before stopping, ensure that all tiered storage partitions
-        # have uploaded at least a manifest: we do not require that they
-        # have uploaded until the head of their log, just that they have
-        # some metadata to validate, so that we will not experience
-        # e.g. missing topic manifests.
-        #
-        # This should not need to wait long: even without waiting for
-        # manifest upload interval, partitions should upload their initial
-        # manifest as soon as they can, and that's all we require.
-
-        def all_partitions_uploaded_manifest():
-            for p in self.partitions():
-                try:
-                    status = self._admin.get_partition_cloud_storage_status(
-                        p.topic, p.index, node=p.leader)
-                except HTTPError as he:
-                    if he.response.status_code == 404:
-                        # Old redpanda, doesn't have this endpoint.  We can't
-                        # do our upload check.
-                        continue
-                    else:
-                        raise
+    def _ssh_output_stderr(self,
```

Comment:
It looks like the original function was quite general, it would be preferable to move it up into somewhere like `util.py` rather than duplicating it.